Pacman.Tests/MovableEntityTests/RandomGhostTests.cs

```diff
@@ -3,10 +3,10 @@
 using System.Linq;
```

Comment:
It took me a short while to realise that the tests are testing both ghost move function and the strategy. Just wonder if it is possible to test strategy by itself without the ghost class.